README.md

```diff
@@ -30,6 +30,41 @@ cd openshift-cluster-management-python-client
 python setup.py install --user
 ```
 
+## Container
+image locate at [openshift-cluster-management-python-wrapper](https://quay.io/repository/myakove/openshift-cluster-management-python-wrapper)
+To pull the image: `podman pull quay.io/myakove/openshift-cluster-management-python-wrapper`
+
+### Examples
+# Usages
+
+```
+podman run quay.io/myakove/openshift-cluster-management-python-wrapper --help
```

Comment:
quay.io/myakove --> replace with quay.io/... (it private)

Reply:
It's not private and we need to put it in our org once we have one